src/Sarif.Multitool.FunctionalTests/TestData/UrisMustBeValid/InvalidUriInOriginalUriBaseIds_Expected.sarif

```diff
@@ -34,8 +34,8 @@
                   "uri": "file:///C:/Code/sarif-sdk/src/Sarif.Multitool.FunctionalTests/TestData/UrisMustBeValid/InvalidUriInOriginalUriBaseIds.sarif"
                 },
                 "region": {
-                  "startLine": 10,
-                  "startColumn": 51
+                  "startLine": 11,
```

Comment:
>startLine [](start = 19, length = 9)

conversion of originalUriBaseIds values from string to file location introduces a new line in the file, hence changes in offsets